src/vivarium_nih_us_cvd/constants/results.py

```diff
@@ -18,7 +18,7 @@
 
 OUTPUT_INPUT_DRAW_COLUMN = "input_data.input_draw_number"
 OUTPUT_RANDOM_SEED_COLUMN = "randomness.random_seed"
-OUTPUT_SCENARIO_COLUMN = "scenario"
+OUTPUT_SCENARIO_COLUMN = "intervention.scenario"
```

Comment:
What's this about?

Reply:
I just reverted a change I had made because I originally made the poc_scenarios file incorrectly